EventHubs/tests/test_eventhub_metrics.py

```diff
@@ -1,48 +1,101 @@
+import sys
 import unittest
-import datetime
+import time
+from io import StringIO
+from datetime import timedelta, datetime
 from baseeventhubtest import BaseEventHubTest
-
+from azure.monitor.query import LogsQueryClient, LogsQueryStatus
+from azure.mgmt.loganalytics import LogAnalyticsManagementClient
 
 class TestEventHubMetrics(BaseEventHubTest):
 
     def setUp(self):
         super(TestEventHubMetrics, self).setUp()
-        self.RESOURCE_GROUP_NAME = "TestEventHubMetrics-%s" % (
-            datetime.datetime.now().strftime("%d-%m-%y-%H-%M-%S"))
-        self.function_name_prefix = "EventHubs_Metrics"
```

Comment:
Convert EventHubs_Metrics as function_name variable and use it in queries.

Reply:
updated